ext/fs/mod.js

```diff
@@ -57,11 +57,55 @@ function removeDirectory(path, recursive) {
 	return core.ops.op_remove_dir(path, recursive);
 }
 
+/**
+ * Copies file asynchronously
+ * @param {string} origin
+ * @param {string} dest
+ * @returns {Promise<void>}
+ */
+function copyFile(origin, dest) {
+	return core.ops.op_copy_file(origin, dest);
+}
+
+/**
+ * Copies folder asynchronously
+ * @param {string} origin
+ * @param {string} dest
+ * @returns {Promise<void>}
+ */
+function copyDirectory(origin, dest) {
+	return core.ops.op_copy_dir(origin, dest);
+}
+
+/**
+ * Moves file asyncrhonously
+ * @param {string} origin
+ * @param {string} dest
+ * @returns {Promise<void>}
+ */
+function moveFile(origin, dest) {
+    return core.ops.op_move_file(origin, dest);
+}
+
+/**
+ * Moves folder asyncrhonously
+ * @param {string} origin
+ * @param {string} dest
+ * @returns {Promise<void>}
+ */
+function moveFolder(origin, dest) {
+    return core.ops.op_move_folder(origin, dest);
+}
+
```

Comment:
@Im-Beast given that these basically call the same op, should this just be `move`?

Reply:
I agree, it doesn't make sense to have two different functions which do exactly the same thing.
I also wonder if `rename` is not a better name, since that's how `std::fs` method is named